project.py

```
@@ -1,18 +1,49 @@
+            for f in files:
+                if f.lower().endswith(".pdf"):
+                    acc.append(os.path.join(dp, f))
+        return sorted(acc)
+    return [root] if root.lower().endswith(".pdf") else []
 
+def main():
+    parser = argparse.ArgumentParser(description="PaperSlicer (WIP)")
+    parser.add_argument("path", nargs="?", help="PDF file or folder")
+    parser.add_argument("--out", help="Output .json or .jsonl")
+    parser.add_argument("--jsonl", action="store_true", help="Write JSON Lines format")
+    args = parser.parse_args()
```

Ruff (0.12.2)

Local variable `args` is assigned to but never used

Remove assignment to unused variable `args`

(F841)